ui/onboarding/Login.qml

```diff
@@ -186,14 +178,25 @@ Item {
             //% "Generate new keys"
             text: qsTrId("generate-new-keys")
             anchors.top: txtPassword.bottom
-            anchors.topMargin: 26
+            anchors.topMargin: 16
```

Comment:
It's better to use the Theme values, but it's not a big deal
```suggestion
            anchors.topMargin: Style.current.padding
```

Reply:
> It's better to use the Theme values, but it's not a big deal

Right, this is the better solution.